### PR TITLE
 few tests added

### PR DESCRIPTION
**Pull request by quasisamurai**
A few tests, that completely pass. May be useful for CI

### CODE REVIEW

File: test/market.js [hunk 1]
```diff
@@ -23,10 +23,11 @@ const IdentityLevel = {
...
 contract('Market', async function (accounts) {
-    let market;
-    let token;
-    let supplier = accounts[1];
-    let consumer = accounts[2];
+    var market;
+    var token;
+    var supplier = accounts[1];
+    var consumer = accounts[2];
+    var master = accounts[3];
```
Comment (Sokel) [inline test/market.js]: seriously?)

File: test/market.js [hunk 2]
```diff
@@ -102,8 +103,40 @@ contract('Market', async function (accounts) {
         // TODO: get balance and check block for 1 hour
 
     });
+    it('test CreateOrder spot ask', async function () {
```
Comment (Sokel) [inline test/market.js]: test naming is duped

File: test/market.js [hunk 3]
```diff
@@ -129,29 +162,43 @@ contract('Market', async function (accounts) {
 
     });
 
-    it('test OpenDeal', async function () {
-        // var deal = await market.OpenDeal(1, 2, {from: consumer});
-        // console.warn(deal);
+    it('test OpenDeal forward', async function () {
+         var deal = await market.OpenDeal(1, 2, {from: consumer});
```
Comment (Sokel) [inline test/market.js]: assert this state

File: test/market.js [hunk 4]
```diff
@@ -129,29 +162,43 @@ contract('Market', async function (accounts) {
...
+         console.warn(deal);
     });
 
-    it('test CloseDeal', function () {
-
+    it('test OpenDeal spot', async function () {
+         var deal = await market.OpenDeal(3, 4, {from: master});
```
Comment (Sokel) [inline test/market.js]: this too

File: test/market.js (hunk 2 above)
Comment (quasisamurai) [reply test/market.js]: fixed

File: test/market.js (hunk 1 above)
Comment (quasisamurai) [reply test/market.js]: yup

**Pull request merged**